src/server/index.js

```diff
@@ -33,23 +33,36 @@ export function Server({ games, db, transport }) {
   }
   transport.init(app, games);
 
-  const api = createApiServer({ db, games });
+  const api = singlePort
+    ? addApiToServer({ app, db, games })
+    : createApiServer({ db, games });
 
   return {
     app,
     api,
     db,
 
-    run: async (port, callback) => {
+    run: async (port, callback, apiPort) => {
       await db.connect();
-      let apiServer = await api.listen(port + 1);
-      let appServer = await app.listen(port, callback);
-      logger.info('listening...');
+
+      let apiServer;
+      if (!singlePort) {
+        apiServer = await api.listen(
+          apiPort ? apiPort : port ? port + 1 : null
```

Comment:
It might be safer to wrap port/apiPort in Number(), because if it's a string, like an env var, it will end up appending the character 1. Which is not the intent.

Reply:
I thought about smth like Number.isInteger check. But maybe wrap is also good. One more fix in this PR :-)